cfme/tests/containers/test_views_objects.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import toolbar as tb
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.5"),
+    pytest.mark.usefixtures('setup_provider')]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+def test_pods_views():
+    sel.force_navigate('containers_pods')
+    tb.select('Grid View')
+    assert tb.is_active('Grid View'), "Pods grid view setting failed"
```

Comment:
Im always torn between copy-paste and extracting to a helper method (which makes the tests a little less readable)
but seeing the same 6 lines each time I think we can extract it 
I would also consider soft assert
and +1 on tier comment - please mark it according to the defined priority